target/linux/ath79/config-6.6

```diff
@@ -169,15 +157,12 @@ CONFIG_PCI_DOMAINS=y
 CONFIG_PCI_DRIVERS_LEGACY=y
 CONFIG_PERF_USE_VMALLOC=y
 CONFIG_PGTABLE_LEVELS=2
-CONFIG_PHYLIB=y
-CONFIG_PHYLIB_LEDS=y
```

Comment:
When is the phylib not needed on ath79? 
If we need an option in all scenarios on a SoC we should compile it into the kernel becasue that saves some space compared to building it as a module.

Reply:
Added phylib back